crates/wash-cli/src/down/mod.rs

```diff
@@ -111,6 +116,7 @@ pub async fn handle_down(cmd: DownCommand, output_kind: OutputKind) -> Result<Co
         cmd.ctl_seed,
         cmd.ctl_credsfile,
         cmd.ctl_tls_ca_file,
+        cmd.ctl_tls_first.unwrap_or(false),
```

Comment:
```suggestion
        cmd.ctl_tls_first.unwrap_or_default(),
```

NIT: Feel free to make this change if you like/prefer it in other places as well!